test/README.md

```diff
@@ -17,13 +17,29 @@ a list of cells to ignore for each notebook, which is useful when specific cells
 intentionally erroneous. Each notebook is then loaded in a browser page, and all its cells are
 executed, and tests pass if no errors are produced.
 
-Tests were written as a nodejs package, so in order to run them locally, you can do `npm install`
-then `npm test` (or `npm install-test`). The one dependency is you have to be able to install
+## Running Tests
+
+### TL;DR
+
+Make sure you can install cairo on your system. Check the `.travis.yml` file for more info
+`cd test`
+`npm install-test`
```

Comment:
I think you should get rid of this npm target. It is only ever useful once, and it's not worth the mental overhead of worrying about whether it is any different than "npm install" followed by "npm test". If you have separate sections for one-time setup and for ongoing running of tests, then the setup section will say to run "npm install" and the test section will say to run "npm test", and the word "install-test" will never appear anywhere.